install_msw.sh

```diff
@@ -127,6 +133,17 @@ then
     git config --global core.fileMode false
 fi
 
+if [ "WSL" = "$platform" ]
+then
+    # Make symbolyc links to lmi and cache_for_lmi directories.
+    # Unfortenely sudo must be used because we don't have permissions to
```

Comment:
```suggestion
    # Unfortunately sudo must be used because we don't have permissions to
```

Also, it's not clear what does it mean "permissions to $directory", you probably mean "permissions to write to $directory"?